eng/Versions.props

```diff
@@ -146,5 +146,6 @@
     <Tooling_MicrosoftVisualStudioLanguageServicesPackageVersion>3.9.0-3.20604.2</Tooling_MicrosoftVisualStudioLanguageServicesPackageVersion>
     <XunitAnalyzersPackageVersion>0.10.0</XunitAnalyzersPackageVersion>
     <XunitVersion>2.4.1</XunitVersion>
+    <XunitCombinatorialVersion>1.4.1</XunitCombinatorialVersion>
```

Comment:
```suggestion
    <XunitCombinatorialPackageVersion>1.4.1</XunitCombinatorialPackageVersion>
```

I believe we're using the `PackageVersion` suffix convention in this file, granted it seems the above `XunitVersion` is missing this as well.

Reply:
➡️ Fixed